fmn/sse/subscriber.py

```diff
@@ -44,23 +51,38 @@ def write_messages_all_connections(self, key):
         if payload:
             self.logger.info(payload)
             connections = self.get_connections(key=key)
+            sse_msg = self.format_msg_sse(msg=payload)
             for req in connections:
-                self.logger.info(req)
-                self.push_sse(payload, req)
+                self.push_sse(sse_msg, req)
+            self.logger.info("Total Connections: " + str(len(connections)))
```

Comment:
Yea that makes sense. This was only necessary to make sure all the connections were coming through during a performance benchmark